libs/providers/flagd-web/src/lib/options.ts

```diff
@@ -0,0 +1,53 @@
+export interface Options {
+  /**
+   * The domain name or IP address of flagd.
+   *
+   * @default window
+   */
+  host: string;
+
+  /**
+   * The port flagd is listen on.
+   *
+   * @default 443
+   */
+  port: number;
+
+  /**
+   * Determines if TLS should be used.
+   *
+   * @default false
+   */
+  tls: boolean;
+
+  /**
+   * Sets maximum delay between connection retries in ms.
+   *
+   * @default 60000
+   */
+  maxDelay: number;
+
+  /**
+   * Sets the maximum number of retries for a connection to be made to the flagd instance
+   * 0 means unlimited.
+   *
+   * @default 0
+   */
+  maxRetries: number;
```

Comment:
Can retries be disabled? Perhaps by using something like `-1`?

Reply:
-1 works to disable retries. I've added documentation and a test for it.